src/pages/journey/apply.tsx

```diff
@@ -1,3 +1,4 @@
+
 import Button from "<@>/components/common/Button";
```

Comment:
components name should be changed to apply

Reply:
The component should changed, and capitalized

Reply:
- can you elaborate on what the first comment is about , commponet should change . And if you can can you ellaborate as well what Aklile rote as well
-